node/repo/fsrepo.go

```diff
@@ -267,11 +270,27 @@ func (fsr *fsLockedRepo) stillValid() error {
 	return nil
 }
 
+func (fsr *fsLockedRepo) verifyDatastoreVlog(p string) error {
+	err := filepath.Walk(p, func(_ string, info os.FileInfo, err error) error {
+		if !info.IsDir() && path.Ext(filepath.Join(p, info.Name())) == "vlog" && info.Size() > MaxSize {
+			er := os.Remove(filepath.Join(p, info.Name()))
```

Comment:
Are you sure you want to automatically remove it? Wouldn't it be better to maybe gzip it and rename it?

Reply:
I mean I don't know whats in the file, and I've had similar issues but still 🤔